create_release_pr/action.yml

```
@@ -0,0 +1,70 @@
+name: Automate creating the Release pull request
+description: Automate creating the Release pull request
+inputs:
+  version:
+    description: Current release version
+    required: true
+  token:
+    description: The GitHub PAT for this action to use
+    required: false
+    default: ${{ github.token }}
+  base_branch:
+    description: The base branch for the release pull request, e.g., "main"
+    required: false
+    default: main
+  changelogConfiguration:
+    description: The changelog configuration file path, e.g., ".github/workflows/config/changelog-release.json"
+    required: true
+  assignee:
+    description: The assignee for the Release pull request, e.g., bot-nimble
+    required: false
+    default: 'bot-nimble'
+  label:
+    description: 'The label for the Release pull request, e.g., "type : release"'
+    required: false
+    default: 'type : release'
+
+runs:
+  using: composite
+  steps:
+    - name: Find HEAD commit
+      id: head
```

how about detect_head_commit or find_head_commit

```suggestion
      id: detect_head_commit
```

---

`find_head_commit` would be good one 👍